assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -430,6 +430,10 @@ che.infra.kubernetes.pvc.precreate_subpaths=true
 # Defines the name of Persistent Volume Claim for che workspaces.
 che.infra.kubernetes.pvc.name=claim-che-workspace
 
+# Defines the storage class of Persistent Volume Claim for the workspaces.
+# Empty strings means "use default".
+che.infra.kubernetes.pvc.storage_class_name=
```

Comment:
@monaka Could you please contribute the corresponding changes to helm chart, to be able to configure `storage_class_name` with helm parameter? Thanks.

Reply:
I agree it's better.
At the same, I have some additional patches to Helm charts `deployments.yaml` and `configmap.yaml`.
So I want to send them (including storage_class_name) as the individual pull request.